### PR TITLE
[LC-726] Edit logic to prevent confirmation of the previous block if it has a mismatch round with the current round of the node.

### PR DESCRIPTION
**Pull request by hyerim-kim**
- Edit logic to prevent confirmation of the previous block if it has a mismatch round with the current round of the node.

### CODE REVIEW

**Comment by hyerim-kim** [comment]
retest please

**Comment by hyerim-kim** [comment]
please retest

**Comment by hyerim-kim** [comment]
retest this please

**Comment by hyerim-kim** [comment]
retest this please

File: loopchain/blockchain/exception.py [hunk 1]
```diff
@@ -125,6 +125,12 @@ class AnnounceNewBlockError(Exception):
     message_code = message_code.Response.fail_announce_block
 
 
+class RoundMismatchError(Exception):
+    """
+    """
+    pass
+
+
```
Comment (metalg0su) [inline loopchain/blockchain/exception.py]: I strongly agree with you to define exceptions. But I think that the name has no similarity between other exceptions (`BlockVersionNotMatch`, BlockHeightMismatch`, ...).

How about consider setting a standard Exception names, and fit the class to it?

**Review by metalg0su — APPROVED** [review]

**Review by windies21 — APPROVED** [review]

**Review by windies21 — APPROVED** [review]

**Pull request merged**